prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/FHIRBundleHelpersTests.kt

```diff
@@ -176,8 +199,8 @@ class FHIRBundleHelpersTests {
         val provenance = bundle.entry.first { it.resource.resourceType.name == "Provenance" }.resource as Provenance
         val outs = provenance.target
         val receiversOut = outs.map { (it.resource as Endpoint).identifier[0].value }
-        assert(receiversOut.isNotEmpty())
-        assert(receiversOut[0] == "co-phd.full-elr-hl7")
+        assertThat(receiversOut.isNotEmpty())
+        assertThat(receiversOut[0] == "co-phd.full-elr-hl7")
     }
```

Comment:
This one should be assertThat(receiversOut[0]).isEqualTo("co-phd.full-elr-hl7")

Reply:
Assertions updated.